src/main/java/com/saralein/server/Constants.java

```diff
@@ -11,6 +11,7 @@ private Constants() {}
 
     public static final Map<Integer, String> STATUS_CODES = Collections.unmodifiableMap(new HashMap<Integer, String>() {{
```

Comment:
Pretty soon I think we will want move the status codes into its own class. You might look into using `Enums` for that.